src/components/PullRequestList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { getPullsAPIPath, getRowsFromPullData } from '../services/api-utils';
+
+const PullRequestList = (): React.ReactElement => {
+  const columns = ['Title', 'Author', 'Status', 'Assignee', 'Link'];
+  const [rows, setRows] = React.useState<string[][]>([]);
+
+  React.useEffect(() => {
+    getPullsAPIPath('openshift', 'console').then((res) =>
+      res.json().then((data) => setRows(getRowsFromPullData(data))),
+    );
+  }, []);
```

Comment:
Perhaps we can extract this into a custom hook that takes in repo name and returns the list of PRs?
I'm imagining something similar to the k8s hooks we have. WDYT?

Reply:
@rottencandy we have this function, `getPullsAPIPath(user, repo)` we can make use of this in future PR. This is just to have a very basic implementation. WDYT?

Reply:
Right. I forgot this PR was only to create a simple service to make a basic API call.

Looks good in that case.